### PR TITLE
Update 1.5

### PR DESCRIPTION
**Pull request by MuellerSeb**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,49 +11,56 @@ source:
...
 
-
 build:
-  number: 1002
-  script: python -m pip install --no-deps --ignore-installed .  --verbose
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - set SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [win]
+    - {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (rth) [inline recipe/meta.yaml:19]:
```suggestion
    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment (MuellerSeb) [reply recipe/meta.yaml:19]: This doesn't work. A lesson I've learned here: https://github.com/conda-forge/gstools-feedstock/commit/37bdbcdcf981f885c8037edd9689134d65ee97a3
This only works for `noarch` builds.

Comment (rth) [reply recipe/meta.yaml:19]: Aww OK.

**Comment by MuellerSeb** [comment]
Yeah, first try! @rth please merge.

**Comment by rth** [comment]
Thanks @MuellerSeb !

**Pull request merged**